content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -112,26 +112,31 @@ Incoming transactions with unclear purposes may be an inconvenience for financia
 
-You can protect against unwanted incoming payments for non-XRP currencies by not creating trust lines in those currencies. Since XRP does not require trust, the `asfDisallowXRP` flag is used to discourage users from sending XRP to an account. However, this flag is not enforced in `rippled` because it could potentially cause accounts to become unusable. (If an account did not have enough XRP to send a transaction that disabled the flag, the account would be completely unusable.) Instead, client applications should disallow or discourage XRP payments to accounts with the `asfDisallowXRP` flag enabled.
+You can protect against unwanted incoming payments for non-XRP currencies by not creating trust lines in those currencies. Since XRP does not require trust, the `asfDisallowXRP` flag is used to discourage users from sending XRP to an account. However, this flag is not enforced in the XRP Ledger protocol because it could potentially cause accounts to become unusable if they run out of XRP. Instead, client applications should disallow or discourage XRP payments to accounts with the `asfDisallowXRP` flag enabled.
 
-You also have the option to block all incoming checks, NFToken offers, payment channels, and trust lines, using these account flags:
+If you want to block _all_ incoming payments, you can enable [Deposit Authorization](depositauth.html). This prevents any transaction from sending money to you, even XRP, unless your account is below the [reserve requirement](reserves.html).
 
-- `asfDisallowIncomingCheck`
-- `asfDisallowIncomingNFTOffer`
-- `asfDisallowIncomingPayChan`
-- `asfDisallowIncomingTrustline`
+If the [DisallowIncoming amendment][] :not_enabled: is enabled, you also have the option to block all incoming Checks, NFTokenOffers, Payment Channels, and trust lines. It is generally harmless to be on the receiving end of these objects, but they can block you from deleting your account and it can be confusing to have objects you didn't expect mixed in with the list of objects you created. To block incoming objects, use one or more of these account flags:
+
+- `asfDisallowIncomingCheck` - for Check objects
+- `asfDisallowIncomingNFTOffer` - for NFTokenOffer objects
+- `asfDisallowIncomingPayChan` - for PayChannel objects
+- `asfDisallowIncomingTrustline` - for RippleState (trust line) objects
+
+When a transaction would create one of these ledger entries, if the destination account has the corresponding flag enabled, the transaction fails with the result code `tecNO_PERMISSION`. Unlike Deposit Authorization, these settings do not prevent you from receiving payments in general. Also, enabling this setting doesn't stop you from creating these types of objects yourself (unless the destination of your transaction is also using the setting, of course).
 
-If a destination account has these flags enabled, `rippled` won't create the corresponding object on the ledger and returns the error code `tecNO_PERMISSION`.
```

Comment:
My rewording here serves a few purposes:

- Doesn't talk about `rippled`.
- Slightly more specific about the situation that prompts this to happen.
- Contrasts with Deposit Authorization since I added a mention of that above.